sneck/Deck.py

```diff
@@ -394,6 +516,86 @@ def update(self, card: dict) -> None:
         
         print(f'[UPDATE]: Updated card {self.__id}.')
 
+    def serialize(self) -> dict:
```

Comment:
I don't know if this function is yet to be completed or not, but just in case this function should provide a complete representation of the object too in case we might want to perform CI tests or save Deck data to file in the future. Maybe something like this might work:

```python
def serialize(self, complete: bool = False):
    result = {'title': self.__title, 'type': "plain" if self.__type == 0 else "text", 'order': self.__order, 'description': self.__description, 'duedate': self.__card_due_time}

    if(complete):
        result | {'more_stuff': 'here'}

    return result
```

This syntax is apparently only possible sine Python 3.9 ([PEP 584](https://www.python.org/dev/peps/pep-0584/)) but it should not be a problem since we can choose whatever version of python we please as the project is yet to be released.